src/main/modules/ardoq/ArdoqClient.ts

```diff
@@ -0,0 +1,93 @@
+import { ArdoqComponentCreatedResponse } from './ArdoqComponentCreatedResponse';
+import { Dependency } from './Dependency';
+
+import axios from 'axios';
+
+export class ArdoqClient {
+  apiKey: string;
```

Comment:
might want to slap a `private` on these, I think it defaults to `public` worth checking though

Reply:
actually, you probably want to define them in your constructor

Reply:
https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/